runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -1304,6 +1307,30 @@ private void addSwaps()
 		{
 			menuManager.addPriorityEntry(new GrimyHerbComparableEntry(this.swapGrimyHerbMode, client));
 		}
+
+		switch (this.swapHopToMode)
+		{
+			//.removePriorityEntry needed here to reset Priority
+			case FRIENDSLIST:
+				menuManager.removePriorityEntry("message");
+				menuManager.addPriorityEntry("hop-to").setPriority(1);
+				menuManager.addPriorityEntry("kick user").setPriority(100);
```

Comment:
I thought the option for kicking was "Kick"? That's what world hopper, hiscores, client scripts, and chat translation seem to use